src/selfplay/game.cc

```diff
@@ -25,10 +25,12 @@ namespace lczero {
 
 namespace{
 const char* kReuseTreeeStr = "Reuse the node statistics between moves";
+const char* kResignPctStr = "Resign when win percentage drops below n";
 }
 
 void SelfPlayGame::PopulateUciParams(OptionsParser* options) {
   options->Add<BoolOption>(kReuseTreeeStr, "reuse-tree") = true;
+  options->Add<FloatOption>(kResignPctStr, 0, 100, "resignpct", 'r')  = 0.0f;
```

Comment:
Maybe `--resign-percentage`? `pct` is a but cryptic, and generally we separate words with hyphens (although tempdecay-moves is an exception)

Reply:
Done